src/main/java/Entities/UserDataClasses/HideableUserDataClasses/Attributes.java

```diff
@@ -19,6 +19,10 @@ public class Attributes extends HideableData<HashMap<Integer, ArrayList<Object>>
     public Attributes(){
         super(new HashMap<Integer, ArrayList<Object>>(), false);
         this.data = super.data;
+        new AttributesDict();
+        for(int i: AttributesDict.attributesMap.keySet()){
+            this.addAttribute(i, 0, true);
+        }
```

Comment:
This is a good solution to prevent bugs in future code. It seems like you're creating an empty "default" constructor for Attributes so that attributes.data is never an empty HashMap, which could cause errors in future code. In an ideal case we would probably implement an option for users to not select an attribute for themselves, but this simple fix works.

Reply:
Thanks homie